WcaOnRails/spec/mailers/job_failure_mailer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe JobFailureMailer, type: :mailer do
+  describe "notify_admin_of_job_failure" do
+    let(:job) { Delayed::Job.new(id: 42) }
+    let(:exception) do
+      begin
+        raise "error!"
+      rescue Exception => e
+        return e
+      end
+    end
+    let(:mail) { JobFailureMailer.notify_admin_of_job_failure(job, exception) }
```

Comment:
Aha. So that's how Ruby exceptions are different from Java exceptions!

You _could_ do `RuntimeError.new("error!").tap{|e| e.set_backtrace("MyTrace")}`  but I understand if you prefer the natural way.

I have a big aversion against throwing exceptions, because it's so "expensive". But I'm not at all sure that's valid for Ruby in 2016.